### DISCUSSION TITLE
Show forecast value from past data

### DISCUSSION THREAD

**Discussion by deryansyah** [Nixtla/statsforecast#632] [Q&A] [answered]
Hello, I'm Sandy, actually I'm new in python, currently exploring the Nixtla multiple model for many series.

Nixtla is very good library, I already implemented the code from End to End Walkthrough so I can get data for next 2024.

What I want to know, How to get the forecast data that running from the first of my data for example if my data start from 1st January 2020 to 31st Dec 2022 and want to forecast from 1st January 2023 to 31st Dec 2023 but I want to include the forecast result from 1st January 2020 to 31st Dec 2022 too.

Thank you

**Comment by jmoralez** [comment, marked as answer]
Hey @deryansyah, thanks for using statsforecast. I believe what you're looking for is [forecast_fitted_values](https://nixtla.github.io/statsforecast/src/core/core.html#statsforecast.forecast_fitted_values).
